src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -998,6 +1001,16 @@ public void WriteLine(string message = null)
                     }
                 }
             }
+            public void WriteLine(string prefix, string message)
```

Comment:
Consider an `int indent` instead of `string prefix`?

Reply:
We could, but overall it'd be doing a string creation and allocation that way so kept it simple :)